optionals/usaf/config.cpp

```diff
@@ -4,12 +4,12 @@ class CfgPatches {
     class ADDON {
         name = COMPONENT_NAME;
         requiredVersion = REQUIRED_VERSION;
-        units[] = {};
-        weapons[] = {};
+        units[] = { };
+        weapons[] = { };
         magazines[] = {};
         requiredAddons[] = {
-            "kat_misc",
-            "rhsusf_main"
+            "kat_acemisc",
+            "rhs_usaf"
```

Comment:
these are the old names? why should we change this to a older name we dont use anymore?